pkg/kubectl/cmd/rollout/rollout_status.go

```diff
@@ -90,7 +103,7 @@ func NewCmdRolloutStatus(f cmdutil.Factory, streams genericclioptions.IOStreams)
 		Run: func(cmd *cobra.Command, args []string) {
 			cmdutil.CheckErr(o.Complete(f, args))
 			cmdutil.CheckErr(o.Validate(cmd, args))
-			cmdutil.CheckErr(o.Run())
+			cmdutil.CheckErr(o.Run(f))
```

Comment:
That's unacceptable, we're reworking all the commands to get rid of this and you're adding it back. If you need dynamic client create it `Complete` and use in `Run`. Never do this!

Reply:
didn't know that, seemed like you've just removed unused argument in this case

I think you meant to comment on https://github.com/kubernetes/kubernetes/pull/67817, will fix it there